generators/src/main/java/peoplegenerator/CookieEmailGenerator.java

```diff
@@ -0,0 +1,56 @@
+package peoplegenerator;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvValidationException;
+
+import java.io.*;
+import java.sql.Timestamp;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+
+public class CookieEmailGenerator {
+    Random random = new Random();
+
+    private String generateTimestamp() {
+        int year = 14 + random.nextInt(5);
+        long start = Timestamp.valueOf("20" + year + "-01-01 00:00:00").getTime();
+        long end = Timestamp.valueOf("20" + (year + 1) + "-01-01 00:00:00").getTime();
+        long diff = end - start;
+        return new Timestamp(start + (long)(Math.random() * diff)).toString();
+    }
```

Comment:
Should we take it out as utils function since we're using it in more than one generator (PageClickGenerator)?